x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/page.tsx

```diff
@@ -81,7 +87,12 @@ export const Page: FC = () => {
           <UpgradeWarning />
 
           <EuiPageContent>
-            <DataFrameAnalyticsList blockRefresh={blockRefresh} />
+            <AnalyticsNavigationBar selectedTabId={selectedTabId} />
+
+            {selectedTabId === 'data_frame_analytics' && (
+              <DataFrameAnalyticsList blockRefresh={blockRefresh} />
```

Comment:
On the Stack Management pages, the jobs list is showing up empty:

![image](https://user-images.githubusercontent.com/7405507/90508688-11c55600-e150-11ea-97cc-4ac783bd0886.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/74978/commits/e5e10d145b79b4da1e05fb5e65b21d06b601503f